composer.json

```diff
@@ -34,6 +34,7 @@
         "openeuropa/oe_multilingual": "~1.0",
         "openeuropa/oe_paragraphs": "~1.3.0",
         "openeuropa/oe_search": "~1.0",
+        "openeuropa/oe_webtools": "dev-OPENEUROPA-2558",
```

Comment:
Use composer patching instead of this.

Reply:
I'll patch after oe_webtools PR is merged.

Reply:
You can still patch with this branch and after just change to master.